test/utils/utils.test.js

```diff
@@ -188,7 +188,7 @@ describe('Utils', () => {
       window.dispatchEvent(event);
       await waitForElement('#milo');
       expect(document.getElementById('milo')).to.exist;
-    })
+    });
```

Comment:
1 : The correction to add a semicolon at the end of the statement is a good practice in Javascript, which can help prevent potential bugs related to automatic semicolon insertion (ASI). It also increases readability.